src/Sinch/SinchClient.cs

```diff
@@ -100,7 +132,7 @@ public class SinchClient : ISinch
             Numbers = new Numbers.Numbers(projectId, new Uri("https://numbers.api.sinch.com/"),
                 _loggerFactory, httpCamelCase);
 
-            // only three regions are available for single-account model. it's eu, us and br. 
+            // only three regions are available for single-account model. it's eu and us. 
```

Comment:
Then there are only 2 regions, right?

Reply:
What is "zt stack" ?
If there are only 2 regions, then you can update the comment accordingly. It's confusing otherwise.

Reply:
Need to create other pr for that about rerouting to supported hosting regions as EU and US